<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUserForm</class>
 <widget class="QWidget" name="NewUserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new user</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameTxt"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="colorTxt">
       <property name="inputMask">
        <string notr="true">\#HHHHHH</string>
       </property>
       <property name="text">
        <string notr="true">#FFFFFF</string>
       </property>
       <property name="placeholderText">
        <string notr="true">#FFFFFF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorBtn">
       <property name="text">
        <string>Pick Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="openImageBtn">
       <property name="text">
        <string>Select Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="imagePreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
